lisa/tests/kernel/scheduler/eas_behaviour.py

```diff
@@ -476,7 +476,7 @@ class TwoBigThreeSmall(EASBehaviour):
     @classmethod
     def get_rtapp_profile(cls, te):
         # 50% of the smallest CPU's capacity
-        small_duty = cls.unscaled_utilization(cls.min_cpu_capacity(te), 50)
+        small_duty = cls.unscaled_utilization(cls.min_cpu_capacity(te), 25)
```

Comment:
Would be nice to explain why that change is done (IIRC it's to match the values used in master).

Reply:
The reason is to make it smaller. That is indeed closer to what was in master, although it cannot be equal, since it now depends on the board. I cannot really see any fundamental reason why EAS would be allowed to not give the expected result for 50, so it's more to check that there are less issues with that value than another. At the end of the day, both are arbitrary, and there is a good chance the test would be supposed to pass with both all the time.

Reply:
I have 0 issue with the change, I know why we added it - I just meant this kind of change should be explained in the commit ("This brings the test behaviour in line with what we have in lisa/master" would be enough).